cmake/Platform/Common/Install_common.cmake

```
@@ -848,7 +848,25 @@ function(ly_setup_assets)
 
                     # copy the empty CMakeList.txt into the gem root directory, to allow add_subdirectory
                     # calls to succeed on the gem root in the install layout
-                    file(CONFIGURE OUTPUT "${gem_scratch_binary_dir}/CMakeLists.txt" CONTENT [[@cmake_copyright_comment@]] @ONLY)
+                    
+
+                    
+
+                        # 1. Create the base CMakeLists.txt that will just include a cmake file per platform
+                        string(CONFIGURE [[
+                    @cmake_copyright_comment@
+                    o3de_read_json_key(GEM_TYPE ${CMAKE_CURRENT_SOURCE_DIR}/gem.json "type")
+                    if (GEM_TYPE STREQUAL "Asset")
+                        include(Platform/${PAL_PLATFORM_NAME}/platform_${PAL_PLATFORM_NAME_LOWERCASE}.cmake)
+                    endif()
+                    ]] subdirectory_cmakelist_content @ONLY)
+                    
+
```

Can you fix the whitespacing here?
```suggestion

                    # 1. Create the base CMakeLists.txt that will just include a cmake file per platform
                    string(CONFIGURE [[
                    @cmake_copyright_comment@
o3de_read_json_key(GEM_TYPE ${CMAKE_CURRENT_SOURCE_DIR}/gem.json "type")
if (GEM_TYPE STREQUAL "Asset")
    include(Platform/${PAL_PLATFORM_NAME}/platform_${PAL_PLATFORM_NAME_LOWERCASE}.cmake)
endif()
]] subdirectory_cmakelist_content @ONLY)
                    
```